erpnext/manufacturing/doctype/bom/bom.json

```diff
@@ -575,15 +578,34 @@
   {
    "fieldname": "scrap_items_section",
    "fieldtype": "Section Break",
+   "hide_border": 1,
    "label": "Scrap Items"
+  },
+  {
+   "default": "0",
+   "fieldname": "fg_based_operating_cost",
+   "fieldtype": "Check",
+   "label": "FG based Operating Cost"
```

Comment:
If with operations is enabled then this should be hide

Reply:
done